.clang-tidy-new

```diff
@@ -0,0 +1,34 @@
+Checks: 'cert-*,
```

Comment:
I used this configuration to fix issues. Since all of directories are not fixed this file not used for now in CI